dashboard/public/frequency/frequency.html

```diff
@@ -4,6 +4,8 @@
 
 <script type="text/javascript" src="/frequency/frequency.js"></script>
 
+<div id="bubble"></div>
```

Comment:
We probably should put the mini instructions somewhere. I always forget
about that.  For frequency we could make an instructions tab similar to
netsim. For vigenere we could put text at the top like text compression?

On Thursday, September 24, 2015, Elijah Hamovitz <notifications@github.com>
wrote:

> In dashboard/public/frequency/frequency.html
> <https://github.com/code-dot-org/code-dot-org/pull/4173#discussion_r40381736>
> :
>
> > @@ -4,6 +4,8 @@
> >
> >  <script type="text/javascript" src="/frequency/frequency.js"></script>
> >
> > +<div id="bubble"></div>
>
> Hmm. Because both of these apps are build around the idea of working with
> some amount of input text, I decided for both of them to put the
> non-markdown instructions in as the "default" text. I suppose there is the
> open question of how valuable it is to provide them with something they can
> click on to show the markdown instructions again.
>
> @sfilman <https://github.com/sfilman>?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/4173/files#r40381736>.
>